cli/cmd/install/install.go

```diff
@@ -114,7 +85,7 @@ type Install struct {
 	HTTPProxy       string   `desc:"Set HTTP_PROXY environment variable for control plane"`
 	Yaml            bool     `desc:"Only print out k8s yaml manifest"`
 	Check           bool     `desc:"Only check status, don't deploy controller"`
-	Lite            bool     `desc:"Only install lite version of Rio(monitoring will be disabled, will be ignored if --disable-features is set)"`
+	Lite            bool     `desc:"Only install lite version of Rio istio install(only works if mesh-mode is istio, monitoring will be disabled, will be ignored if --disable-features is set)"`
```

Comment:
do we need to prevent the users from `rio install --lite --mesh-mode linkerd` and `rio install --lite` or will this silently install "full" when mesh mode is not `istio` ? @StrongMonkey

Reply:
@izaac it will ignore lite if using linkerd and install full linkerd. Let’s keep it right now as cli may get changed after cli review.